src/Carousel/Carousel.st.css

```
@@ -86,8 +87,39 @@
   left: 0;
 }
 
+.root:showEdgesGradient :global(.container) {
+  padding: 40px;
+  background: rgba(0, 0, 0, 1);
+}
+
+.root:showEdgesGradient :global(.slick-list) {
+  position: relative;
+}
+
+.root:showEdgesGradient :global(.slick-list:before),
+.root:showEdgesGradient :global(.slick-list:after) {
+  content: "";
+  position: absolute;
+  z-index: 1;
+  width: 100px;
+  top: 0;
+  height: 100%;
+  pointer-events: none;
+}
+
+.root:showEdgesGradient :global(.slick-list:before) {
+  left: 0;
+  background: linear-gradient(to left, rgba(240, 244, 247, 0) 50%, var(--edgesGradientColor, value(D70)));
```

For backwards compatibility define the same property twice with and without the css variable, like so:
```suggestion
  background: linear-gradient(to left, rgba(240, 244, 247, 0) 50%, value(D70));
  background: linear-gradient(to left, rgba(240, 244, 247, 0) 50%, var(--edgesGradientColor, value(D70)));
```